### ISSUE TITLE
Feature wanted: Multiple namespaces for a group

### ISSUE THREAD

**Issue by rbatllet** [luckylooke/dragular#9]
I'll need that a group could belong to more then one namespace. 

Imagine that we have fruits, vegetables and meat groups, then an apple must be dropped into the fruits or into the vegetables group.

**Comment by luckylooke** [comment]
I was looking at the possible implementation but it seems to have performance affect also for users who dont need such feature, so I decided not to implement it. You can make desired behavior via classes. Simply add classes to draggable item like:
```html
<div class="apple fruit red"> item1 </div>
```
and while initiating containers provide custom accepts function
```js
dragularService(document.getElementsByClassName('container'), {
   accepts: function(item){
       item = angular.element(item);
       return item.hasClass('fruit') || item.hasClass('red');
   }
});
```

**Comment by luckylooke** [comment]
It depends on setup but in same cases you will need also isContainer function to make it work correctly
```
dragularService(document.getElementsByClassName('container'), {
   accepts: function(item){
       item = angular.element(item);
       return item.hasClass('fruit') || item.hasClass('red');
   },
   isContainer: function(elm){
       return.angular.element(elm).hasClass('container');
    }
});
```

**Comment by luckylooke** [comment]
I have come with solution which is not slowing down other users, at least not those who dont use namespacing. I was testing it with my machine and even using multiple namespaces do not cause visible pref issues. So it can be merged. I will add it in 1.5.0. Additional perf tweaks are possible via replacing angular.forEach with native js loops. I am planing to do that later.

**Comment by luckylooke** [comment]
If you need it now, it is in [dev brach v150](https://github.com/luckylooke/dragular/tree/v150).

**Comment by luckylooke** [comment]
fixed in release 2.0.0
